### PR TITLE
[SystemSettings][TCSACR-215] Add properties

### PR DESCRIPTION
**Pull request by jinwangan**
- AccessibilityGrayscale
 - AccessibilityNegativeColor

Signed-off-by: jinwang.an <jinwang.an@samsung.com>

### CODE REVIEW

File: src/Tizen.System.SystemSettings/Tizen.System.SystemSettings/SystemSettingsEventArgs.cs [hunk 1]
```diff
@@ -1120,4 +1120,70 @@ public bool Value
...
+        private readonly bool _accessibilityGrayscale;
+        internal AccessibilityGrayscaleChangedEventArgs(bool val)
+        {
+            _accessibilityGrayscale = val;
+        }
+
+        /// <summary>
+        /// Indicates whether developer option state is enabled on the device or not.
+        /// </summary>
+        /// <since_tizen> 5.5 </since_tizen>
```
Comment (m-macias) [inline src/Tizen.System.SystemSettings/Tizen.System.SystemSettings/SystemSettingsEventArgs.cs]: Please change the API version to 6

File: src/Tizen.System.SystemSettings/Tizen.System.SystemSettings/SystemSettingsEventArgs.cs [hunk 2]
```diff
@@ -1120,4 +1120,70 @@ public bool Value
...
+        private readonly bool _accessibilityNegativeColor;
+        internal AccessibilityNegativeColorChangedEventArgs(bool val)
+        {
+            _accessibilityNegativeColor = val;
+        }
+
+        /// <summary>
+        /// Indicates whether developer option state is enabled on the device or not.
+        /// </summary>
+        /// <since_tizen> 5.5 </since_tizen>
```
Comment (m-macias) [inline src/Tizen.System.SystemSettings/Tizen.System.SystemSettings/SystemSettingsEventArgs.cs]: Please change the API version to 6

File: src/Tizen.System.SystemSettings/Tizen.System.SystemSettings/SystemSettingsEventArgs.cs (hunk 1 above)
Comment (jinwangan) [reply src/Tizen.System.SystemSettings/Tizen.System.SystemSettings/SystemSettingsEventArgs.cs]: Done

File: src/Tizen.System.SystemSettings/Tizen.System.SystemSettings/SystemSettingsEventArgs.cs (hunk 2 above)
Comment (jinwangan) [reply src/Tizen.System.SystemSettings/Tizen.System.SystemSettings/SystemSettingsEventArgs.cs]: Done

**Pull request merged**